server/config/rateLimit.js

```diff
@@ -0,0 +1,13 @@
+import rateLimit from 'express-rate-limit';
+
+const timeFrameInHours = 24;
```

Comment:
I think 1 hour is better, 24 hours seems to much to me.

```suggestion
const timeFrameInHours = 1;
```